.travis.yml

```diff
@@ -74,7 +74,7 @@ script:
   - bundle exec rake data_updates:run # required for at least creating the first admin user in e2e tests.
   - yarn e2e:ci
   # Dropping and recreating the database for Capybara/rspecs
-  - curl -XDELETE "${ELASTICSEARCH_URL}/*" # resetting elastic search post e2e tests
+  - bundle exec rake search:destroy # resetting elastic search post e2e tests
```

Comment:
Can't remember who suggested this, but this is more resilient as the search implementation is abstracted away.